assets/ovn-monitoring/queries.libsonnet

```diff
@@ -10,49 +10,38 @@ local generateTimeSeriesQuery(query, legend) = [
 ];
 
 {
-  ovnMasterLeader: {
+  ovnClusterManagerLeader: {
     query():
-      generateTimeSeriesQuery('ovnkube_master_leader', '{{pod}}'),
```

Comment:
metric doesn't exist. Replacement is only `_leader` that is unique, as `ovnkube_controller_leader` is 0 for all pods.